src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -337,6 +348,7 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
             .subscribe((model: DotPageContainer[]) => {
                 this.ngZone.run(() => {
                     this.saveContent(model);
+                    this.dotEditContentHtmlService.setContaintersSameHeight(this.pageState.layout);
```

Comment:
Now that we are for the changes in the DOM, do we need to do this where when the ```pageModel``` change? because if the ```pageModel``` change it means the DOM changed.

Reply:
for some weird reason after the model change this is not been called